### PR TITLE
Hw 3 functions

### PR DESCRIPTION
**Pull request by e-hryhorova**
3 tasks on Lesson about Functions. For now I'm missing one task on grouping the dictionary, will add it later when finish

### CODE REVIEW

File: task_3_part_1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+soglasnye = ['b', 'c', 'd', 'f', 'g', 'h', 'k', 'l', 'm', 'n', 'p', 'r', 's', 't', 'w', 'v', 'x', 'z']
```
Comment (mmykhailenko) [inline task_3_part_1.py]: These letters must be stored in one of mutable data types (str() or tuple())
Example:
`"bcdfghjklmnpqrstvwxyz" or (b', 'c', 'd', 'f', 'g', 'h', 'k', 'l', 'm', 'n', 'p', 'r', 's', 't', 'w', 'v', 'x', 'z')`

File: task_3_part_1.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+def change_consonants_letters_to_vowel(s):
+    stroka2 = ""
+    for i in s:
+        if i in soglasnye:
+            letter = 'a'
+            stroka2 = stroka2 + letter
+        else:
+            stroka2 = stroka2 + i
+    return stroka2
```
Comment (mmykhailenko) [inline task_3_part_1.py]: string have method `.replace()` you could use that :)

**Review by mmykhailenko — CHANGES_REQUESTED** [review]

File: task_3_part_1.py (hunk 2 above)
Comment (e-hryhorova) [reply task_3_part_1.py]: Please check, I updated according your comments

**Comment by e-hryhorova** [comment]
Hi, 
I updated first task using module random.
I added another function  to task 3 for grouping a dictionary

File: task_3_part_2.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            list_kiev.append(d)
+        elif d['city'] == 'Dnepr':
+            list_cities.append(d.pop('city'))
+            list_dnepr.append(d)
+        else:
+            list_cities.append(d.pop('city'))
+            list_lviv.append(d)
+    values = [list_kiev, list_dnepr,  list_lviv]
+    result = dict(zip(list_cities, values))
+    return result
```
Comment (mmykhailenko) [inline task_3_part_2.py:37]: очень не обычный подход. Похожее решение одного из ваших коллег но без дополнительных объектов
```
def group_by_city(some_arg):
    result_data = {}
    for item in some_arg:
        city = item.pop('city')
        if city in result_data:
            result_data[city].append(item)
        else:
            result_data[city] = [item]
    return result_data
```

**Comment by mmykhailenko** [comment]
LGTM

**Review by mmykhailenko — APPROVED** [review]

**Pull request merged**